shared/rakefiles/scripts/sync_gke_istio_state.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env sh
+
+# This script goes through a list of Istio HPAs ($ISTIO_HPAS) and tries to
+# import them into Terraform state in given Terragrunt module
+# ($ISTIO_MODULE_DIR). This is intended to sync Terraform state with actual
+# state, as istio components on GKE are managed by Google.
+
+set -eou pipefail
+
+# enable debug output if $DEBUG is set to true
+[ "${DEBUG:=false}" = 'true' ] && set -x
+
+# get script name
+THIS_SCRIPT="$(basename "${0}")"
+
+# list of env variables - interface
+# required
+ENV=${ENV:?"Environment variable must be set"}
+# optional
+ISTIO_MODULE_DIR=${ISTIO_MODULE_DIR:="live/${ENV}/k8s/istio"}
+ISTIO_HPAS=${ISTIO_HPAS:="istio-egressgateway istio-ingressgateway istio-pilot istio-policy istio-telemetry"}
+REQUIRED_BINARIES=${REQUIRED_BINARIES:="kubectl terragrunt jq"}
+
+# check the module does exist, otherwise silently exit
+if [ ! -d "${ISTIO_MODULE_DIR}" ]
+then
+  echo "${THIS_SCRIPT}: Istio module not found (${ISTIO_MODULE_DIR})"
```

Comment:
This message looks like an error. Can you clarify that this situation is not (necessarily) bad?

Reply:
👍, done